packages/elements-dev-portal/src/hooks/useGetNodes.ts

```diff
@@ -23,6 +23,6 @@ export function useGetNodes({
   return useQuery(
     ['workspaceNodes', workspaceId, branchSlug, projectIds, debounceSearch, platformUrl, platformAuthToken],
     () => getNodes({ workspaceId, branchSlug, projectIds, search: debounceSearch, platformUrl, platformAuthToken }),
-    { enabled: !pause ? true : false },
```

Comment:
I'd prefer `pause ? false : true` :kappa:

(Kidding)

Reply:
well that's what it was _originally_... https://github.com/stoplightio/elements/pull/1773#discussion_r712394496

I'm fine with changing it back to that if it's actually easier for y'all to read. Maybe I'm just too old and wizened. @paulatulis @mmiask does `? false : true` it make it clearer that the returned value is a boolean or you like that there's no implicit type casting to remember or...?

Reply:
(I was 100% kidding and think `!pause` is pretty "idiomatic"!)

Reply:
FYI, I frequently would tell Jakub R his code was too terse and hard to read and he should make it “dumber” so I could understand it at a glance, so I’m totally open to receiving the same feedback about my code.